style.css

```diff
@@ -0,0 +1,90 @@
+@import url("https://bermuda-css.netlify.app/style.css");
+@import url("./stylesheets/abstracts/colors.css");
+@import url("./stylesheets/utils/utility.css");
+
+body {
+  background-color: var(--defalult-bg-color);
+  box-sizing: border-box;
+  color: var(--default-text-color);
+}
+.root {
+  height: 100vh;
+  margin: auto;
+  width: 100vw;
+}
+/* content component */
+.content {
+  margin: auto;
+  position: relative;
+  width: 100%;
+}
+/* header comonent styling */
+.header-container {
+  position: sticky;
+  top: 0;
+  z-index: 999;
```

Comment:
can store z-indexes in variables and use, avoid random large numbers.

Reply:
Created Variables for z-index.